index.html

```diff
@@ -0,0 +1,5 @@
+<html>
+  <head>
+    <meta http-equiv="refresh" content="0; url=/torchtune/stable/index.html">
```

Comment:
Re https://github.com/pytorch/torchtune/pull/563/files#r1536130428, would that work? (with a TODO comment saying we'll need to update it when stable is available?)

```suggestion
    <meta http-equiv="refresh" content="0; url=/torchtune/main/index.html">
```